search-index.json

```diff
@@ -0,0 +1,15 @@
+---
+layout: null
+---
+[
+{% for coll in site.collections %}
+{% for doc in coll.docs %}
+{
+  "id": {{ doc.url | jsonify }},
+  "title": {{ doc.title | jsonify }},
+  "text": {{ doc.content | strip_html | strip_newlines | jsonify }}
```

Comment:
`strip_newlines` should not be required as minisearch's tokenizer splits on all punctuation and whitespace anyway.